packages/bpk-tokens/src/react-native-output-test.js

```diff
@@ -44,6 +44,15 @@ const ANDROID_ONLY_PROPS = [
 ];
 
 const IOS_ONLY_PROPS = [
+  'BACKGROUND_ELEVATION_01_DARK_COLOR',
```

Comment:
These aren't IOS_ONLY.
Instead of adding them here, remove them from ANDROID_ONLY above